edward/models/distributions.py

```diff
@@ -81,7 +81,7 @@ def log_prob(self, xs):
         Parameters
         ----------
         xs : tf.Tensor or np.ndarray
-            n_minibatch x self.shape
+            n_samples x self.shape
```

Comment:
In the most generic context (any of use distribution objects), maybe this one should be renamed to `size` similar to `Distribution().sample()`?